agir/events/components/eventPage/EventDescriptionCard.js

```diff
@@ -0,0 +1,166 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import styled from "styled-components";
+
+import { DateTime } from "luxon";
+import Button from "@agir/front/genericComponents/Button";
+import Collapsible from "@agir/front/genericComponents/Collapsible.js";
+import Card from "../../../front/components/genericComponents/Card";
+import Spacer from "../../../front/components/genericComponents/Spacer";
```

Comment:
```suggestion
import Card from "@agir/front/components/genericComponents/Card";
import Spacer from "@agir/front/components/genericComponents/Spacer";
```

Reply:
Désolé, c'est plutôt :
```suggestion
import Card from "@agir/front/genericComponents/Card";
import Spacer from "@agir/front/genericComponents/Spacer";
```